mappings/akn.mapping

```diff
@@ -0,0 +1 @@
+CLASS akn
```

Comment:
This looks like a mistake

Reply:
Yeah, Enigma does this sometimes. Not sure why. There's a few other files like this, so shouldn't be a huge issue. Can remove if needed, though.